src/PriceOracle.sol

```diff
@@ -25,16 +25,18 @@ contract PriceOracle is AccessControl {
     uint64 public lastAdjustedRound;
 
     // The minimum price allowed
-    uint32 public minimumPrice = 1024;
+    uint32 public minimumPrice = 24000;
 
     // The priceBase to modulate the price
-    uint32 public priceBase = 514155;
+    uint32 public priceBase = 524288;
 
     // The current price is the atomic unit.
     uint32 public currentPrice = minimumPrice;
 
+    uint32 private currentPriceUpscaled = minimumPrice << 10; // we upscale it by 2^10
```

Comment:
make it uint64 as you use it in other places.

Reply:
Actually, this is probably not needed uint32 has 10 digits and our currentPriceUpscaled is usually around 8. But we could just give it more safe room for future and less worry.